src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/RazorSemanticTokensInfoService.cs

```diff
@@ -233,7 +224,7 @@ private static ImmutableArray<SemanticRange> CombineSemanticRanges(ImmutableArra
                 {
                     if (colorBackground)
                     {
-                        tokenModifiers |= (int)RazorSemanticTokensLegend.RazorTokenModifiers.razorCode;
+                        tokenModifiers |= (int)RazorSemanticTokensLegendService.RazorTokenModifiers.razorCode;
```

Comment:
Is there a reason to leave these statics on the service? Should they just be pulled into something like `RazorTokenModifiers` as a static class itself?

Reply:
I think at the time I added this, I thought it was just overkill to do the thing we do with token types (ensuring they're after the Roslyn ones numerically) because there is only one modifier, and Roslyn only has two. Your question does really point out how weird/inconsistent this is though, so I will change this to work the same as the tokens.

(its not actually a static, its a nested enum type, so it could also just be moved out of the service, but I would feel wierd about a random enum with one entry hanging around in space)

Reply:
Nope, can't make this the same as the tokens, because the tokens use reflection to get all of the fields, so if i make the modifiers a field, then they'll be included in the tokens 🤦‍♂️ Will have to think about how to best do this, but will do it in a follow up.